kstests-in-cloud.sh

```diff
@@ -39,7 +39,7 @@ usage () {
 $(basename $0) [options] COMMAND TARGET
 
 Run kickstart tests on runners temporarily provisioned by linchpin in cloud.
-Linchpin target name TARGET is defined in PinFile (linchpin/PinFile).
+Linchpin target topology name TARGET is defined in PinFile. See `topology_name` value in the linchpin/PinFile.
 
```

Comment:
TARGET is actually target name. See the PinFile - target consists of topology (with the same name in default PinFile) and layout. Name of the target is defined in PinFile as the top-level element (not topology_name) - see example PinFiles, eg https://github.com/rhinstaller/kickstart-tests/blob/master/linchpin/examples/example1/PinFile which contains target example1. (There can be multiple targets defined in single PinFile).

But I agree I should improve this to make clear what target actually is - a group of resources (test runners) that can be provisioned / used / destroyed by the script, defined in PinFile. And mention that only single instance of target named TARGET can be running on single host at the same time.

Reply:
I've tried to describe this better.

Reply:
"See the value of the top-level element in the linchpin/PinFile for the name" ?

Reply:
Sounds great. Thanks!

Reply:
Fixed.